### PR TITLE
fixed status check

### PR DESCRIPTION
**Pull request by rafaelcidade**

### CODE REVIEW

File: src/services/ethService.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ async function getTxInfo(txHash) {
     value,
     asset,
     contractAddress,
-    status: txReceipt.status === '0x1' ? 'confirmed' : 'failed',
+    status: (txReceipt.status == '0x1') ? 'confirmed' : 'failed',
```
Comment (ricardoul) [inline src/services/ethService.js]: why == and not ===?

File: src/services/ethService.js [hunk 2]
```diff
@@ -791,7 +791,7 @@ async function addERC20(receipt) {
     const decimals = await contract.methods.decimals().call();
     const totalSupply = await contract.methods.totalSupply().call();
 
-    if (receipt.status === '0x1') {
+    if (receipt.status == '0x1') {
```
Comment (ricardoul) [inline src/services/ethService.js]: why == and not ===?

File: src/services/ethService.js [hunk 3]
```diff
@@ -834,7 +834,7 @@ async function addERC721(receipt) {
     const symbol = await contract.methods.symbol().call();
     const name = await contract.methods.name().call();
 
-    if (receipt.status === '0x1') {
+    if (receipt.status == '0x1') {
```
Comment (ricardoul) [inline src/services/ethService.js]: why == and not ===?

File: test.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+// Redis
+const redis = require('redis');
+const redisOptions = {host: process.env.REDIS_SERVER, port: process.env.REDIS_PORT, password: process.env.REDIS_PW};
+try {
+  redis.createClient(redisOptions);
+  logger.info("Successfully connected to Redis server")
+} catch (e) { logger.error(e) }
+bluebird.promisifyAll(redis);
+
```
Comment (ricardoul) [inline test.js]: This file should be included at all?

File: src/services/ethService.js (hunk 1 above)
Comment (rafaelcidade) [reply src/services/ethService.js]: txReceipt.status is Bool.. so it fails type check

**Pull request closed without merging**